ssl/ssl_lib.c

```diff
@@ -1714,14 +1718,45 @@ int SSL_shutdown(SSL *s)
     }
 }
 
-int SSL_renegotiate(SSL *s)
+int SSL_key_update(SSL *s, SSL_KEY_UPDATE updatetype)
 {
     /*
-     * TODO(TLS1.3): Return an error for now. Perhaps we should do a KeyUpdate
-     * instead when we support that?
+     * TODO(TLS1.3): How will applications know whether TLSv1.3+ has been
```

Comment:
Well, that's the discussion we had before!! My compromise was to add this TODO instead of the SSL_is_tls13_plus() function (deferring that to another PR as its more of a tangential issue). And of course we disagree on whether we should be thinking TLSv1.3 or TLSv1.3+....but as its only a TODO comment I guess I can compromise on the compromise. I'll still be thinking TLSv1.3+ when I come to look at this again though ;-)